### PR TITLE
Install of sentry cli in pipeline must return true

### PR DESCRIPTION
**Pull request by lauer**
Ensure that the install of Sentry CLI always returns true, as the example on Freestyle project does

### CODE REVIEW

**Comment by imatwawana** [comment]
@jan-auer can you review?

**Comment by manuzope** [comment]
Hey @lauer, thanks for opening this PR! I'm not very familiar with Jenkins but why would you always want to return `true`?

**Comment by lauer** [comment]
It's only the install cmd. Because that will fail if the cli tool is already installed. 

If the install cmd fails the next cmd would fail anyway.

File: src/docs/product/releases/setup/release-automation/jenkins/index.mdx [hunk 1]
```diff
@@ -63,7 +63,7 @@ pipeline {
             }
             steps {
                 // Install Sentry CLI
-                sh 'curl -sL https://sentry.io/get-cli/ | bash'
+                sh 'curl -sL https://sentry.io/get-cli/ | bash || true'
```
Comment (joshuarli) [inline src/docs/product/releases/setup/release-automation/jenkins/index.mdx:66]:
```suggestion
                command -v sentry-cli || sh 'curl -sL https://sentry.io/get-cli/ | bash'
```

Comment (joshuarli) [inline src/docs/product/releases/setup/release-automation/jenkins/index.mdx:66]: Is this more what you're trying to say? If sentry-cli isn't installed, then we should install it. This won't fail if sentry-cli isn't installed, assuming the installation succeeds.

```suggestion
                sh 'command -v sentry-cli || curl -sL https://sentry.io/get-cli/ | bash'
```

**Review by joshuarli — APPROVED** [review]

**Comment by joshuarli** [comment]
Thanks @lauer!

**Pull request merged**